<launch>
  <node name="camera0_publisher" pkg="simplecamerapublisher" type="simplecamerapublisher_node" output="screen">
    <param name="camera_topic" value="/camera0/image" />
    <param name="sensor_id" value="0" />
    <param name="width" value="1280" />
    <param name="height" value="720" />
    <param name="frame_rate" value="15" />
  </node>
  <node name="camera1_publisher" pkg="simplecamerapublisher" type="simplecamerapublisher_node" output="screen">
    <param name="camera_topic" value="/camera1/image" />
    <param name="sensor_id" value="1" />
    <param name="width" value="1280" />
    <param name="height" value="720" />
    <param name="frame_rate" value="15" />
  </node>
</launch>